### DISCUSSION TITLE
Connection Problem. Yes to pi, no to internet

### DISCUSSION THREAD

**Discussion by tschaki14** [pivpn/pivpn#1405] [Support]
Hello everybody

I'm am very new to all of this and I am sorry if I missed any manual or help thread. But I set up pivpn as instructed and I can connect to my pi via openvpn. I can also use my terminal app to ping others in my LAN. But I am not connected to the internet. Any ideas? Do you need anything for problem solving?

Cheers and thank you!

**Comment by tschaki14** [comment]
Ok I am sorry. After the right googledoodeling I found a solution: 'pivpn debug' showed me some problems with masquerade, ip stuff (dunno what) but saying yes to everything it works!
Found a thread vom 2017 adressing this problem after each reboot. Don't know if this problem still persists after a reboot.

**Reply by coolapso** [reply]
good you found it :)

**Comment by coolapso** [comment]
Hi, welcome, 

here: https://docs.pivpn.io/faq/#how-do-i-troubleshoot-connection-issues 

Also helps reading here: https://github.com/pivpn/pivpn/discussions/1341

**Reply by tschaki14** [reply]
Ou I see. Ehhhhh sorrey! Wrong everything =)
Btw: thanks for all your work..  !!!
